src/components/tabs/tabs.jsx

```diff
@@ -0,0 +1,43 @@
+import React, {Fragment} from "react";
+import PropTypes from "prop-types";
+import {Link} from "react-router-dom";
+import movieCardProp from "../movie-card/movie-card.prop";
+
+const Tabs = (props) => {
+  const {movieData, tabsItems} = props;
+  const {pathname, hash} = window.location;
+  const currentLink = pathname + hash;
+  const [currentTabData] = tabsItems.filter((tabData) => currentLink === tabData.link);
+  const CurrentTabComponent = currentTabData.component;
```

Comment:
Можно тут сразу компонент и доставать:
```suggestion
  const [{ component: CurrentTabComponent }] = tabsItems.filter((tabData) => currentLink === tabData.link);
```